Mathlib/Probability/Kernel/Disintegration/CdfToKernel.lean

```diff
@@ -64,11 +64,11 @@ and for all measurable sets `s : Set β`, `∫ b in s, f (a, b) x ∂(ν a) = (
 Also the `ℚ → ℝ` function `f (a, b)` should satisfy the properties of a Sieltjes function for
 `(ν a)`-almost all `b : β`. -/
 structure IsRatCondKernelCDF (f : α × β → ℚ → ℝ) (κ : kernel α (β × ℝ)) (ν : kernel α β) : Prop :=
-  (measurable : Measurable f)
-  (isRatStieltjesPoint_ae (a : α) : ∀ᵐ b ∂(ν a), IsRatStieltjesPoint f (a, b))
-  (integrable (a : α) (q : ℚ) : Integrable (fun b ↦ f (a, b) q) (ν a))
-  (set_integral (a : α) {s : Set β} (_hs : MeasurableSet s) (q : ℚ) :
-    ∫ b in s, f (a, b) q ∂(ν a) = (κ a (s ×ˢ Iic (q : ℝ))).toReal)
+  measurable : Measurable f
```

Comment:
I had no idea the parentheses around the fields were not needed. That review taught me something. Thanks!